### ISSUE TITLE
Spamfilter makes it impossible to ask questions/write comments for new Users!

### ISSUE THREAD

**Issue by Finchtastic** [opencv-infrastructure/answers.opencv.org#53]
I signed up on http://answers.opencv.org and it took me two days and at least 10 tries to ask a question. All my tries got detected by the spamfilter even tho there was nothing looking like spam. 

Now I have some answers to my question and it is not possible for me to write any comment because of the spam filter. It's literally impossible to use the forum as a new user at the moment.

**Comment by mshabunin** [comment]
@Finchtastic , we use external spam filter and we can not tune any parameters. The statistics dashboard shows 20 spam messages blocked in October. Could you please post the question body and the error page screenshot here, we will take a look and investigate it?

**Comment by Finchtastic** [comment]
@mshabunin  The attachments show the comment and what happened after I clicked "add
comment".
I tried to write another ~10 comments and all got deleted because of spam.
If your counter is right it should be ~30 spam messages now and I seem to
be the only one who gets blocked.
If its still on 20 you have some other problems I guess.

![comment](https://user-images.githubusercontent.com/28004988/31495930-c3d87e0c-af59-11e7-9f81-587829bb8878.jpg)
![error message](https://user-images.githubusercontent.com/28004988/31495931-c3f1dac8-af59-11e7-9224-3e411d1803d5.jpg)

**Comment by mshabunin** [comment]
This is strange, the question seems broken. I can not edit it or post any comments either. Could you please register another account and try to ask the same question one more time?

**Comment by Finchtastic** [comment]
The new account works. I can ask and also comment other questions/answers. 

The old "Finch" account still gets blocked from writing anything.

**Comment by smacl6301** [comment]
Same problem here, I'll try creating a new account

**Comment by smacl6301** [comment]
Created a new account and still have this problem

**Comment by mshabunin** [comment]
@smacl6301 , could you please post the body of your question here? What are your account names?

**Comment by smacl6301** [comment]
Ok, problem sorted after a fashion.  I was using Chrome and the Captcha wasn't working.  Created another new account, this time in explorer and all works fine.  In Chrome, the I'm not a robot Captcha comes back immediately with a green tick to signal success.  In explorer, it goes through a series of picture verification stages first.   Chrome version is 62.0.3202.94 (Official Build) (64-bit), O/S is Windows 7 enterprise 64 bit, SP1.  New user account that works is shanemacl, other accounts are smacl6301 and Shane MacLaughlin.  Text of post is also on Stackoverflow here https://stackoverflow.com/questions/47292078/using-opencv-projectpoints-to-overlay-real-world-data-on-a-photo but seems to be ok as is now awaiting moderation when entered via explorer,

**Comment by berak** [comment]
@smacl6301 

[you're in, as it seems](http://answers.opencv.org/question/178342/using-opencv-projectpoints-to-overlay-real-world-data-on-a-photo/)

**Comment by smacl6301** [comment]
Yes indeed, looks like a Chrome configuration issue of some kind.  No biggie, I'll use explorer for this site.

**Comment by mshabunin** [comment]
I can recommend trying incognito mode, resetting site settings and clearing cookies in the site settings ("i" icon in the address bar).
